### DISCUSSION TITLE
승객 생성 알고리즘

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#18] [Ideas]
현재 승객 생성 알고리즘
- 10초에 한번씩, 전체 역 중 2/3의 역에서 승객이 생성

이 경우 게임이 지나치게 정적으로 보일 우려가 있습니다.
다른 알고리즘에 대한 의견 바랍니다.

**Comment by cstrnull00** [comment]
* 변경
  * 각 역이 활성화 된 시점부터 역마다 틱을 가짐
  * 각 역에서 반나절(일반 속도 기준 6초, 빠른 속도 3초)마다 66% 확률로 승객이 생성
